### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by boulderuphillroller**
typos

### CODE REVIEW

**Review by nik-suri — APPROVED** [review]
Lgtm, thank you!

**Review by bruce-riley — APPROVED** [review]

File: wormchain/x/ibc-hooks/README.md [hunk 1]
```diff
@@ -87,20 +87,20 @@ An ICS20 packet is formatted correctly for wasmhooks iff the following all hold:
 * `memo["wasm"]["msg"]` is a valid JSON object
 * `receiver == "" || receiver == memo["wasm"]["contract"]`
 
-We consider an ICS20 packet as directed towards wasmhooks iff all of the following hold:
+We consider an ICS20 packet as directed towards wasmhooks if all of the following hold:
```
Comment (panoel) [inline wormchain/x/ibc-hooks/README.md:90]: Same as above comment

Comment (boulderuphillroller) [reply wormchain/x/ibc-hooks/README.md:90]: ah good point, thanks!

File: wormchain/x/ibc-hooks/README.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ ICS20 is JSON native, so we use JSON for the memo format.
 }
 ```
 
-An ICS20 packet is formatted correctly for wasmhooks iff the following all hold:
+An ICS20 packet is formatted correctly for wasmhooks if the following all hold:
```
[the remark this answers is not in the document]
Comment (SEJeff) [reply wormchain/x/ibc-hooks/README.md:81]: I agree with @panoel this is meant to be iff.

**Review by SEJeff — COMMENTED** [review]
Nice change thanks. Please fix these small things first.

Did you run a spell checker over the docs or just manually read them all?

**Comment by SEJeff** [comment]
Closing due to inactivity. If you'd like to rebase ontop of the latest and re-open this we can look into merging.

**Pull request closed without merging**